### DISCUSSION TITLE
TSML WP Plugin Ver 3.14.1 update

### DISCUSSION THREAD

**Discussion by DCO84770** [code4recovery/12-step-meeting-list#778] [Q&A] [answered]
Hello
We use ver 3.13. Will updating to latest ver of 3.14.1 alter the template we use for uploading CSV meeting list files?
Thanks
Roger P

**Comment by kiyote33** [comment, marked as answer]
No, version 3.14.1 does not alter the schema used for uploading CSV files at all.  We always recommend updating to the latest, but most local admins find it prudent to test the latest version on a staging site prior to going live, just to be safe.
